src/parser/mod.rs

```diff
@@ -8192,11 +8192,19 @@ impl<'a> Parser<'a> {
     /// Parses a column definition within a view.
     fn parse_view_column(&mut self) -> Result<ViewColumnDef, ParserError> {
         let name = self.parse_identifier(false)?;
-        let options = if dialect_of!(self is BigQueryDialect | GenericDialect)
-            && self.parse_keyword(Keyword::OPTIONS)
+        let mut options = vec![];
+        if (dialect_of!(self is BigQueryDialect | GenericDialect)
+            && self.parse_keyword(Keyword::OPTIONS))
+            || (dialect_of!(self is SnowflakeDialect | GenericDialect)
+                && self.parse_keyword(Keyword::COMMENT))
         {
             self.prev_token();
-            Some(self.parse_options(Keyword::OPTIONS)?)
+            if let Some(option) = self.parse_optional_column_option()? {
+                options.push(option);
+            }
+        };
+        let options = if !options.is_empty() {
+            Some(options)
         } else {
             None
         };
```

Comment:
would it make more sense to rewrite to something like this?
```rust
if (dialect_of!(....)) {
  self.prev_token();
  if let Some(option) = self.parse_optional_column_option()? {
    Some(vec![option])
  } else {None}
} else {
  None
};
```
thinking it helps with the following:
- we avoid the mut option
- the `is_empty` check as well, that one doesn't seem necessary, because the if condition already checks that we have an option

Reply:
Updated.